lib/config.js

```diff
@@ -259,7 +259,7 @@ internals.server = Validate.object({
     })
         .unknown()
         .default(),
-    mime: Validate.object().allow(null).default(null),
+    mime: Validate.object().empty(null).default(),
```

Comment:
This change accounts for the fact that the `Mimos` constructor no longer accepts `null`.